audio/audio.py

```diff
@@ -5,6 +5,16 @@
 from xblock.core import XBlock
 from xblock.fields import Integer, Scope, String
 from xblock.fragment import Fragment
+from xblockutils.resources import ResourceLoader
+
+resource_loader = ResourceLoader(__name__)
+
+
+def _(text):
+    """
+    Make `_` a no-op, so we can scrape strings
+    """
+    return text
```

Comment:
Is this a pattern we're using in other repos? If so, could you link an example?

Reply:
yes, it's a pattern @brian-smith-tcril 

 - https://github.com/openedx/xblock-drag-and-drop-v2/blob/9030c3807fea019370523dd235789c4f09c0d2d0/drag_and_drop_v2/utils.py#L14-L17